### PR TITLE
add documentation

### PR DESCRIPTION
**Pull request by ygnn123**
Add documentation for person-vehicle-bike-detection-crossroad-1016 model

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
@ygnn123 please note that other models description have just empty Performance section. If you do not provide exact values here, it is better to left it empty instead of 'tbd'

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
@ygnn123 please also note that new file should be added to index.md
10:23:07  person-vehicle-bike-detection-crossroad-1016 not in index.md file

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: intel_models/person-vehicle-bike-detection-crossroad-1016/description/person-vehicle-bike-detection-crossroad-1016.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+| Type of object | Number of bounding boxes |
+|----------------|--------------------------|
+| Vehicle        | 810,323                  |
+| Pedestrian     | 1,114,799                |
+| Non-vehicle    | 62,334                   |
+
+## Performance (FPS)
+
+### Performance disclaimer
```
Comment (zmaslova) [inline intel_models/person-vehicle-bike-detection-crossroad-1016/description/person-vehicle-bike-detection-crossroad-1016.md]: please remove `(FPS)`, `### Performance disclaimer` and `### Optimization notice`

**Comment by snosov1** [comment]
Fine by me.

**Comment by zmaslova** [comment]
@ygnn123 also change target branch to develop

File: intel_models/person-vehicle-bike-detection-crossroad-1016/description/person-vehicle-bike-detection-crossroad-1016.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+1. Net outputs blob with shape: [1, 1, N, 7], where N is the number of detected
+   bounding boxes. For each detection, the description has the format:
+   [`image_id`, `label`, `conf`, `x_min`, `y_min`, `x_max`, `y_max`]
+    - `image_id` - ID of the image in the batch
+    - `label` - predicted class ID
+    - `conf` - confidence for the predicted class
+    - (`x_min`, `y_min`) - coordinates of the top left bounding box corner
+    - (`x_max`, `y_max`) - coordinates of the bottom right bounding box corner.
+
+## Legal information
```
Comment (zmaslova) [inline intel_models/person-vehicle-bike-detection-crossroad-1016/description/person-vehicle-bike-detection-crossroad-1016.md]:
```
## Legal Information
[*] Other names and brands may be claimed as the property of others.
```

File: intel_models/person-vehicle-bike-detection-crossroad-1016/description/person-vehicle-bike-detection-crossroad-1016.md (hunk 1 above)
Comment (zmaslova) [reply intel_models/person-vehicle-bike-detection-crossroad-1016/description/person-vehicle-bike-detection-crossroad-1016.md]: you didn't remove `### Performance disclaimer` and `### Optimization notice`...

**Comment by zmaslova** [comment]
Jenkins please retry a build

**Comment by zmaslova** [comment]
Jenkins please retry a build

**Pull request merged**